Langs/1055.lang

```diff
@@ -1,59 +1,59 @@
 [MetaData]
 Language=Turkish - Turkey
 Translator=eren885
-Version=Sürüm
+Version=2.2.5.0
 [Strings]
-Donate= Bağış
+Donate=Bağış
 Check for Updates=Güncellemeleri kontrol et
 Your Windows 11 Compatibility Results are Below=Windows 11 ile uyumluluk sonuçlarınız aşağıdadır
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=WhyNotWin11'e https://www.whynotwin11.org/ ile ulaş!
-Results Based on Currently Known Requirements!=Sonuçlar şu an Microsoft'un erdiği bilgilerle derlenmiştir!
-Translation by=Bu uygulama şu kişi tarafından çevirilmiştir;
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=WhyNotWinn11'e şimdi https://www.whynotwin11.org/ üzerinden ulaş!
+Results Based on Currently Known Requirements!=Sonuçlar, şu anki bilinen gereksinlimlere göre derlenmiştir!
+Translation by=Bu uygulama Türkçe'ye şu kişi tarafından çevrildi;
```

Comment:
No need for that Turkish here, I think it is logical.

Reply:
So I think only `Bu uygulama şu kişi tarafından çevrildi;` will be good, right?

Reply:
yeah